e2e-tests/tests/conftest.py

```diff
@@ -18,3 +19,11 @@ def session_capabilities(pytestconfig, session_capabilities):
     if pytestconfig.getoption('driver') == 'SauceLabs':
         session_capabilities.setdefault('tags', []).append('socorro')
     return session_capabilities
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    if os.environ.get('MOZ_HEADLESS'):
```

Comment:
This will deem any non-empty string as `True`.

If you think that might cause issues (for example, if someone then decided to do `MOZ_HEADLESS=0`), then maybe we should fix this to something like this:

```
if os.environ.get('MOZ_HEADLESS') == '1':
```

If you don't think that's likely to happen, we can leave it as is.